src/main/java/com/marklogic/appdeployer/command/forests/DeployCustomForestsCommand.java

```diff
@@ -56,6 +56,7 @@ protected void processDirectory(File dir, CommandContext context) {
 				logger.info("Processing forests in file: " + f.getAbsolutePath());
 			}
 			String payload = copyFileToString(f, context);
+			adjustPayloadBeforeSavingResource(mgr, context, f, payload);
```

Comment:
This would be a place where we cannot use that "saveResource" method I just suggested above, since it's calling saveJsonForests instead of save.